sys/docker/alpine-nginx/Dockerfile

```diff
@@ -14,12 +14,13 @@ RUN apk add --no-cache --virtual frontenddeps \
     && yarn add sass-loader@^8.0.0 node-sass --dev \
     && npm run build
 
-FROM nginx:1.19-alpine
+FROM fabiocicerchia/nginx-lua:1.19.3-alpine3.12.0
```

Comment:
Is this intended?

Reply:
yes, that's a custom mod for supporting lua (used for incrementing redis counter).